test/signup.js

```diff
@@ -61,14 +62,15 @@ export async function signup(driver) {
     await driver.wait(until.titleIs('Verification - Open Educational Badges'), defaultWait);
 }
 
-export async function deleteUserOverApi(username = 'automated@test.mail', password = 'automatedTestPassword') {
+// Not sure if we still need these functions so I commented them out.
+/* export async function deleteUserOverApi(username = 'automated@test.mail', password = 'automatedTestPassword') {
```

Comment:
I don't like commented out code. Besides, I think it makes sense to delete the user over the API if the deletion via UI fails. Because otherwise, if something is off with our UI, the test will not only fail but also break future tests.

Reply:
I reverted this change.